doc/source/ray-air/examples/torch_image_batch_pretrained.py

```diff
@@ -23,7 +22,8 @@ def preprocess(batch: np.ndarray) -> pd.DataFrame:
             transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
         ]
     )
-    return pd.DataFrame({"image": [preprocess(image) for image in batch]})
+    transformed_images = np.array([preprocess(image).numpy() for image in batch])
+    return transformed_images
```

Comment:
Should we encourage users to create structured datasets instead of tensor datasets? 

```suggestion
    return {"image": transform_images"}
```

No strong opinion, just wondering what the tradeoff is.